app/1.0/docs/devguide/data-binding.md

```diff
@@ -643,92 +565,65 @@ _or_ if the array itself is mutated, so the binding uses a wildcard path, `myArr
 </dom-module>
```

Comment:
For some reason I can't do a line comment on line 544 but there's an extra space before the closing ] and an extra carriage return on 541